docs/source/advanced_tutorial/updating_projects.rst

```diff
@@ -30,6 +30,6 @@ Click "Update Project" after each icon click. Click "OK" when done.
 Refresh Outputs
 ===============
 
-If you made changes to either the `Building Characteristics Report <https://github.com/NREL/OpenStudio-BuildStock/tree/master/measures/BuildingCharacteristicsReport>`_ or `Simulation Output Report <https://github.com/NREL/OpenStudio-BuildStock/tree/master/measures/SimulationOutputReport>`_ measures, you will need to make updates on the Outputs tab. To do this remove the measure(s) from the project and re-add them, while making sure they occupy the original position in the workflow order. Then navigate to the Outputs tab, select the measure(s) you've re-added, and then choose the outputs to include. Click "OK" when done. These outputs will be included in the summary results csv file.
+If you made changes to either the `Building Characteristics Report <https://github.com/NREL/OpenStudio-BuildStock/tree/master/measures/BuildingCharacteristicsReport>`_ or `Simulation Output Report <https://github.com/NREL/OpenStudio-BuildStock/tree/master/measures/SimulationOutputReport>`_ measures, you will need to make updates on the Outputs tab. To do this remove the measure(s) from the project and re-add them, making sure they occupy the original position in the workflow order. If you changed any ``housing_characteristics`` tsv filenames, you will additionally need to make a change (e.g., add a whitepace) to ``measure.rb`` of `Building Characteristics Report <https://github.com/NREL/OpenStudio-BuildStock/tree/master/measures/BuildingCharacteristicsReport>`_. Then navigate to the Outputs tab, select the measure(s) you've re-added, and then choose the outputs to include. Click "OK" when done. These outputs will be included in the summary results csv file.
```

Comment:
@joseph-robertson We should copy or link to the second half of this from the section on modifying probability distributions. https://resstock.readthedocs.io/en/latest/advanced_tutorial/modifying_probability_distributions.html

Reply:
@ejhw Take a look at my addition.